src/components/Expander/Expander.jsx

```diff
@@ -50,6 +49,22 @@ const Expander = createClass({
 				`,
 			},
 		}),
+		AdditionalLabelContent: createClass({
+			displayName: 'Expander.AdditionalLabelContent',
+			statics: {
+				peek: {
+					description: `
+						Renders a \`<span>\` to be shown next to the expander label.
+					`,
+				},
+			},
+			propName: 'AdditionalLabelContent',
+			propTypes: {
+				children: node`
```

Comment:
thoughts on this being a render prop function over a node?

Reply:
I don't think having this as a function would add any value, since we don't need `Expander` to inject anything into what we're rendering here.
We just want to provide content for `Expander` to place next to its label.